tests/acceptance/CarouselHeaderClassicCept.php

```diff
@@ -44,7 +44,7 @@
 $I->amOnPage( '/' . $slug );
 
 // Check the Carousel header block.
-$I->see( 'Header 1', '.carousel-captions-wrapper > h1' );
+$I->see( 'Header 1', '.carousel-captions-wrapper > h2' );
```

Comment:
Why don't you use the same description as you've added in the line `60`?
I mean this `$I->see( 'Header 2', '.carousel-captions-wrapper > h2' );`

Reply:
@GP-Dan-Tovbein Can you elaborate why it would need to be the same?

Reply:
What's the difference between this [check](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/797/files#diff-6e06a6fd5f39d252bb07be764214a1cac379ea6d8c336953fa0f0f74be0091edR47) and this other [check](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/797/files#diff-6e06a6fd5f39d252bb07be764214a1cac379ea6d8c336953fa0f0f74be0091edR60)? Is there sth that I'm missing?